lib/Serialization/ModuleFile.cpp

```diff
@@ -321,6 +321,34 @@ void ModuleFile::lookupValue(DeclName name,
   }
 }
 
+void ModuleFile::lookupSwiftValueByObjCName(
+    DeclName name, SmallVectorImpl<ValueDecl *> &results) {
+  if (Core->TopLevelDecls) {
```

Comment:
Curious how significant the performance penalty of iterating all declarations is / how difficult it is to alter the serialization format to include a table by objc name as well.